app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -732,6 +738,30 @@ public boolean onTouch(View v, MotionEvent event) {
     }
 
     /**
+     * Saves the current Info Window location to shared preferences.
+     * @param appraisalWindowPosition Current Info Window Y offset for appraisal mode.
+     */
+    private void saveWindowPosition(int appraisalWindowPosition) {
+        SharedPreferences.Editor edit = sharedPref.edit();
+        edit.putInt("appraisalWindowPosition", appraisalWindowPosition);
```

Comment:
Preference key names should be constants, like for all other ones, and not just for consistency: On code following this convention it's easier to spot typos. And it's easier to find other uses of the same key by navigating through users of the constants.